templates/music/album_detail.html

```diff
@@ -4,6 +4,37 @@
 {% load markdown_deux_tags %}
 {% load music %}
 
+{% block socialmeta %}
+    <!-- Google+ data -->
+    <meta itemprop="name" content="{{ album.name }} by {{ artist.name|upper }}">
```

Comment:
`artist` isn't in the context here, so this will show up as "Album XYZ by ".
You can instead use `{{ album.project.artist }}` to access the artist.

You're also mixing cases here. Are you sure you want to have the artist name in uppercase?